src/terrain/clients/analyses.clj

```diff
@@ -0,0 +1,40 @@
+(ns terrain.clients.analyses
+  (:require [terrain.util.config :refer [analyses-base-uri]]
+            [clj-http.client :as http]
+            [cemerick.url :refer [url]]
+            [terrain.auth.user-attributes :refer [current-user]]
+            [slingshot.slingshot :refer [throw+]]))
```

Comment:
I think that `[slingshot.slingshot :refer [throw+]]` can be removed now unless I'm missing something. I didn't see `throw+` used anywhere.

Reply:
This should be fixed now.